### ISSUE TITLE
Duplicate errors being written to the DB table

### ISSUE THREAD

**Issue by thomasgatt** [dereuromark/CakePHP-DatabaseLog#18]
Running CakePHP v3.3.16 I've noticed a very strange issue with logging manually (i.e. using Log::write() from a controller or a view)

It seems that logs are being written to the database twice. After several testing I've noticed that the issue is due to the config in the app.php file;

'Log' => [
		'debug' => [
			'className' => 'DatabaseLog.Database'
		],
		'error' => [
			'className' => 'DatabaseLog.Database'
		],
	],

With the above config, the logged will save two records in the table. But the following config will save only one record:
'Log' => [
		'debug' => [
			'className' => 'DatabaseLog.Database'
		]
	],

**Comment by dereuromark** [comment]
It is most likely because you didnt the scopes on the main ones:
http://sandbox.dereuromark.de/pages/best-practices#logging

**Comment by thomasgatt** [comment]
Should I disable the scopes in the app.php file? I am following your setup code:

> You can simply modify the existing config entries in your config/app.php:
> 
>    'Log' => [
>    	'debug' => [
>    		'className' => 'DatabaseLog.Database'
>    	],
>    	'error' => [
>    		'className' => 'DatabaseLog.Database'
>    	],
>    ],

**Comment by dereuromark** [comment]
try it. helped me.

**Comment by thomasgatt** [comment]
I get a RunTime error - Could not load class debug.

But with the following code everything works perfectly
```
'Log' => [
		'debug' => [
			'className' => 'DatabaseLog.Database',
		],
	],
```

**Comment by dereuromark** [comment]
Maybe the example isnt clear, you should only add

    'scopes' => false,

into it, not completely replace the config..

**Comment by thomasgatt** [comment]
Yes of course. This produces duplicate logging:

```
	'Log' => [
		'debug' => [
			'className' => 'DatabaseLog.Database',
			'scopes' => false,
		],
		'error' => [
			'className' => 'DatabaseLog.Database',
			'scopes' => false,
		]
	],
```

If I add another log level, the log is written 3 times instead of twice. But if I leave only one log level (debug) for example, I get only one. Basically scopes => false doesn't seem to affect. Could be a bug that was fixed in versions > 3.3.16?

**Comment by dereuromark** [comment]
Of course, this is a more recent fix for sure, you should always be using the latest 3.x version, always.
